Ruby/Ruby.sublime-syntax

```diff
@@ -748,6 +748,12 @@ contexts:
     - match: '\)'
       scope: meta.function.parameters.ruby punctuation.definition.group.end.ruby
       pop: true
+    - match: '(?=#)'
+      scope: comment.line.number-sign.ruby
```

Comment:
why not just `- include: comments` as opposed to having a `scope` that isn't used, a `push` and an unnecessary `with_prototype`?